config.dev.yaml

```diff
@@ -9,3 +9,13 @@ services:
   scraper:
     endpoint: "/errors"
     refresh_interval: 10s
+- name: periskop
+  dns_sd_configs:
+  - names:
+    - localhost
+    refresh_interval: 10s
+    type: A
+    port: 7777
+  scraper:
+    endpoint: "/errors"
+    refresh_interval: 10s
```

Comment:
now that we support more service discovery mechanisms, we can use a static config for this instead of DNS:

```
services:
- name: periskop
  static_configs:
  - targets:
    - localhost:7777
  scraper:
    endpoint: "/errors"
    refresh_interval: 10s
```

maybe leave the mock target as DNS so we can showcase two diferent types of SD in the example config